web/components/groups/group-overview.tsx

```diff
@@ -0,0 +1,369 @@
+              </p>
+            </div>
+          )}
+          {pinned.map((element, index) => (
+            <div className="relative my-2">
+              {element}
+
+              {editMode && (
+                <div>
+                  <button
+                    className=" text-gray-500 hover:text-gray-700"
+                    onClick={() => {
+                      console.log(group.pinnedItems)
+                      const newPinned = group.pinnedItems.filter((item) => {
+                        return item.itemId !== group.pinnedItems[index].itemId
+                      })
+                      updateGroup(group, { pinnedItems: newPinned })
+                    }}
+                  >
+                    <div className="absolute top-0 left-0 right-0 bottom-0 bg-gray-200 bg-opacity-50">
+                      <XCircleIcon className="h-12 w-12 text-gray-600" />
+                    </div>
+                  </button>
+                </div>
+              )}
+            </div>
+          ))}
+          {editMode && group.pinnedItems && pinned.length < 6 && (
+            <div className=" py-2">
+              <Row
+                className={
+                  'relative gap-3 rounded-lg border-4 border-dotted p-2 hover:cursor-pointer hover:bg-gray-100'
+                }
+              >
+                <button
+                  className="flex w-full justify-center"
+                  onClick={() => setOpen(true)}
+                >
+                  <PlusCircleIcon
+                    className="h-12 w-12 text-gray-600"
+                    aria-hidden="true"
+                  />
+                </button>
+              </Row>
+            </div>
+          )}
+        </Masonry>
+      </div>
+      <PinnedSelectModal
+        open={open}
+        group={group}
+        posts={posts}
+        setOpen={setOpen}
+        title="Pin a post or market"
+        description={
+          <div className={'text-md my-4 text-gray-600'}>
+            Pin posts or markets to the overview of this group.
+          </div>
+        }
+        onSubmit={onSubmit}
+      />
+    </>
+  ) : (
+    <></>
+  )
+}
+
+function SectionHeader(props: {
+  label: string
+  href?: string
+  children?: ReactNode
+}) {
+  const { label, href, children } = props
+  const content = (
+    <>
+      {label}{' '}
+      <ArrowSmRightIcon
+        className="mb-0.5 inline h-6 w-6 text-gray-500"
+        aria-hidden="true"
+      />
+    </>
+  )
+
+  return (
+    <Row className="mb-3 items-center justify-between">
+      {href ? (
+        <SiteLink
+          className="text-xl"
+          href={href}
+          onClick={() => track('group click section header', { section: href })}
+        >
+          {content}
+        </SiteLink>
+      ) : (
+        <span className="text-xl">{content}</span>
+      )}
+      {children}
+    </Row>
+  )
+}
+
+export function GroupAbout(props: {
```

Comment:
This function was just moved (cut and pasted) from the group index page, no new code.

Reply:
Thanks for letting me know!